Description: Provide missing definiton of \+ command
 doxygen's tex output contains a \+ command, which would normally
 be defined in the standard doxygen header, but srtp is using a
 custom header which lacks this definition.
 This patch adds the missing definition, following the approach
 used to solve https://bugs.debian.org/751355 .
Author: Si Padmore <sipadmore@gmx.co.uk>
Last-Update: 2014-08-22
Bug-Debian: http://bugs.debian.org/752970

--- srtp-1.4.5~20130609~dfsg.orig/doc/header.template
+++ srtp-1.4.5~20130609~dfsg/doc/header.template
@@ -77,6 +77,7 @@
 \fontfamily{ppl}
 \selectfont
 
+\newcommand{\+}{\discretionary{\mbox{\scriptsize$\hookleftarrow$}}{}{}}
 
 \renewcommand{\familydefault}{\sfdefault}
 \begin{document}
